### DISCUSSION TITLE
Black Sreen But...

### DISCUSSION THREAD

**Discussion by karnaKali** [hyprwm/Hyprland#8600] [General]
I'm on Arch 6.12.1-2-cachyos kernal. Everything was working perfectly until I updated yesterday. The crazy thing is I used my keybinding to open the terminal and typed reboot and it did? .It's working but I can't see anything? I don't know what to try.

Thanks in advance.

**Comment by gulafaran** [comment]
mesa broke intel in 24.3 , most likely that. https://gitlab.freedesktop.org/mesa/mesa/-/issues/12195 , downgrade

**Reply by karnaKali** [reply]
That was it! I thank you soo much. I search and searched but never saw that page.
